web/src/components/ParticipantForm/Participant.css

```diff
@@ -0,0 +1,71 @@
+/* Mobile Layout */
```

Comment:
Can you add a screenshot of how it looks on mobile?

Reply:
Sure! But, mobile version is incomplete. That will be one of my next issues to solve. 
![123](https://user-images.githubusercontent.com/44326790/88121524-9346ba00-cb8b-11ea-9c9b-1f2e80523f94.PNG)